bindata/configmaps/console-public-configmap.yaml

```diff
@@ -1,3 +1,5 @@
+# This configmap 'console-public' manifest is used to expose the console URL
+# to all authenticated users
```

Comment:
Since we are moving this manifest from the `/manifest` (which would be created the CVO) and instead will be created and handled by console-operator, we should remove all the `include.release.openshift.io` and `release.openshift.io` annotations.